searchcore/src/vespa/searchcore/proton/flushengine/flushengine.h

```diff
@@ -79,7 +79,7 @@ class FlushEngine final : public FastOS_Runnable
     uint32_t initFlush(const IFlushHandler::SP &handler, const IFlushTarget::SP &target);
     void flushDone(const FlushContext &ctx, uint32_t taskId);
     bool canFlushMore(const std::unique_lock<std::mutex> &guard) const;
-    bool wait(vespalib::duration minimumWaitTimeIfReady);
+    bool wait(vespalib::duration minimumWaitTimeIfReady, bool considerPendingPrune);
```

Comment:
This is called _ignorePendingPrune_ in the cpp file.

Reply:
Fixed in #16298